webapp/components/wpt-results.html

```diff
@@ -374,36 +393,18 @@ <h3>
         };
       }
 
-      async fetchResults(testRuns) {
-        if (!testRuns) {
-          return;
+      // fetchResults should be triggered by changes in searchURL or diffRun,
+      // but diffRun is only used in subroutine: refreshDisplayedNodes.
```

Comment:
I don't see where `diffRun` is loaded now.

Reply:
`fetchDiff` is invoked here: https://github.com/web-platform-tests/wpt.fyi/pull/431/files/343ee0a740cb4a7c46a28e262f076b6584a886cd#diff-ff3585b170ed88df8ef0c2fa8bcc61ecR383

It sets `diffRun` here https://github.com/web-platform-tests/wpt.fyi/pull/431/files/343ee0a740cb4a7c46a28e262f076b6584a886cd#diff-ff3585b170ed88df8ef0c2fa8bcc61ecR415 which is observed here https://github.com/web-platform-tests/wpt.fyi/pull/431/files/343ee0a740cb4a7c46a28e262f076b6584a886cd#diff-ff3585b170ed88df8ef0c2fa8bcc61ecR313